packages/charts/src/vaadin-chart.js

```diff
@@ -1781,6 +1789,34 @@ class Chart extends ResizeMixin(ElementMixin(ThemableMixin(PolymerElement))) {
   __showWarn(propertyName, acceptedValues) {
     console.warn('<vaadin-chart> Acceptable values for "' + propertyName + '" are ' + acceptedValues);
   }
+
+  /** @private */
+  __onRedraw() {
+    this.__checkTurboMode();
+  }
+
+  /** @private */
+  __checkTurboMode() {
+    const isDevelopmentMode = !!window.Vaadin.developmentMode;
+
+    if (!this.configuration || !isDevelopmentMode || this.__turboModeWarningAlreadyLogged) {
+      return;
+    }
+
+    const exceedsTurboThreshold = this.configuration.series.some((series) => {
+      const threshold = (series.options && series.options.turboThreshold) || 0;
```

Comment:
I added the null checks and the `0` fallback only as a safe-guard because I don't know the Highcharts library too well. From my testing it seemed that `series.options.turboThreshold` is always defined, but I wasn't sure if that would always be the case.